megamek/src/megamek/common/Infantry.java

```diff
@@ -1497,6 +1497,16 @@ public boolean isSquad() {
         return squadCount == 1;
     }
 
+    @Override
+    public boolean isEligibleForPavementBonus() {
+        if ((game != null)
```

Comment:
Is this supposed to apply to battle armor too?

Reply:
no, it only applies to wheeled, tracked, and motorized infantry. Rule first sentence ``If mechanized and motorized infantry...``

Someone on discord did ask if this and the main TW rule should apply to hover as well. I wouldn't think so, but it _IS_ a ground vehicle. Currently MM doesn't allow for a hover CV/SV to get the pavement bonus so that would require a rules question to sort out.

Reply:
Hover should get it. TW pg 22 defines ground vehicles and hover is included.

Reply:
Ok, that will be also fix a longstanding unknown bug in Tank.java then. It does not have hover listed.